.werft/util/uninstall-gitpod.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+NAMESPACE=$1
+
+if [[ -z ${NAMESPACE} ]]; then
+   echo "One or more input params were invalid. The params we received were: ${NAMESPACE}"
+   exit 1
+fi
+
+echo "Removing Gitpod in namespace ${NAMESPACE}"
+kubectl get configmap gitpod-app -n "${NAMESPACE}" -o jsonpath='{.data.app\.yaml}' | kubectl delete -f -
```

Comment:
This shellscript is necessary because the `kubectl delete` was getting a no resources found in the TypeScript file. In other words, this is a workaround.

Reply:
Understood. Would be awesome if the installer had such a `delete` feature itself.

Reply:
Hey @geropl what would the use case be for that? I ask because we also have `with-clean-slate-deployment` - which removes either Helm or Installer, and then installs with the Installer if no `with-helm` was specified. If `with-helm` was specified, it'll clean then install with Helm.

Reply:
Or do you mean have the installer connect to the Kubernetes installation, and remove Gitpod itself?

Reply:
> Or do you mean have the installer connect to the Kubernetes installation, and remove Gitpod itself?

Sorry for not being clear: yes. IMO the installer should be able to completly remove everything it added. We already have the `gitpod-app` Configmap which contains everything required (although only for the last `render` run).